docker-compose.yml

```diff
@@ -14,10 +11,15 @@ services:
       REDIS_PASSWORD: ${REDIS_PASSWORD}
       AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
       AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
+      ENV: development
```

Comment:
The idea of a development, test, and production environment is a Rails thing. What's the distinction that it provides here?